packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+:::
+
+## Prerequisites
+
+Before adding a peer, you and the account servicing entity you intend to peer with must both:
+
+- Run an implementation of an [Interledger connector](/integration/deployment/services/backend-service#interledger-connector) (ideally Rafiki).
+- Agree on an [asset](/overview/concepts/accounting#assets) for the peering relationship. You can set up multiple peering relationships with the same peer based on different assets. At least one asset shared by you and your peer must be added to your Rafiki instance prior to setting up the peering relationship. For more information, refer to [Assets](/integration/requirements/assets/).
+- Agree on a `maxPacketAmount` value, which specifies how many packets a payment is split into.
+- Communicate your respective static <LinkOut href="https://interledger.org/developers/rfcs/ilp-addresses/">Interledger (ILP) addresses</LinkOut>.
+- Communicate a connection endpoint for the other peer to send packets to.
+- Exchange `auth` tokens for the connection endpoint.
+- Agree on a settlement mechanism (a means of paying one another for the successful forwarding and delivery of packets), which is outside the scope of Interledger and Rafiki.
+
+Optionally, you may also deposit an initial liquidity for your peer.
+
+:::note
+Peering is not required unless you want to participate in transactions with another ASE running Rafiki. For foundational peering concepts, refer to [Peers](/overview/concepts/interledger/#peers).
+:::
+
+## Setting up peering in Rafiki
+
+The basic workflow of setting up a peering relationship starts with adding the agreed upon asset, then adding a peer, and then creating a wallet address from where you can send and receive payments. 
+
+### Add an asset
+
+As mentioned in the prerequisites, an asset must be added to your Rafiki instance before creating a peering relationship. To learn how to add an asset, refer to [Assets](/integration/requirements/assets/).
+
+### Add a peer
+
+After adding an asset, add a peer using the `createPeer` GraphQL mutation.
+
+#### `createPeer`
+
+<CodeBlock title="Example createPeer Mutation">
+
+```graphql
+mutation CreatePeer($input: CreatePeerInput!) {
+  createPeer(input: $input) {
+    code
+    success
+    message
+    peer {
+      id
+      asset {
+        code
+        scale
+      }
+      staticIlpAddress
+      name
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+##### Example
+
+<CodeBlock title='Example JSON request'>
+
+```json
+{
+  "input": {
+	"staticIlpAddress": "g.othergreatwallet",
+	"name": "The Other Great Wallet"
+	"http": {
+  		"incoming": {"authTokens": ["mytoken"]},
+  		"outgoing": {"endpoint": "ilp.othergreatwallet.com", "authToken": "theirtoken"}
+	  },
+	"assetId": "INSERT_ASSET_ID",
+	"initialLiquidity": <optionally, an intial amount of liquity to provision. Liquidity can also be deposited via the `DepositPeerLiquidity`>
```

Comment:
Small thing: Either change end to "deposited via DepositPeerLiquidity" or "deposited via the DepositPeerLiquidity mutation"

Reply:
Fixed